src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -1,18 +1,12 @@
 package seedu.addressbook.commands;
 
-import java.util.HashSet;
-import java.util.Set;
-
 import seedu.addressbook.data.exception.IllegalValueException;
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
-import seedu.addressbook.data.person.ReadOnlyPerson;
-import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.*;
```

Comment:
Imported classes should always be listed explicitly. E.g. `java.util.List` and not `java.util.*`, which does not follow NUS Java coding standards.